home.html

```diff
@@ -0,0 +1,190 @@
+                <span class="bar"></span>
+            </div>
+        </div>
+        <div class="mobile-menu">
+            <ul>
+                <li><a href="home.html" onclick="closeMenu()">Home</a></li>
+                <li><a href="about.html" onclick="closeMenu()">About</a></li>
+                <li><a href="#" onclick="closeMenu()">Program</a></li>
+                <li><a href="#" onclick="closeMenu()">Join</a></li>
+                <li><a href="#" onclick="closeMenu()">Sponsor</a></li>
+                <li><a href="#" onclick="closeMenu()">News</a></li>
+            </ul>
+        </div>
+        <div class="miniNav">
+            <div class="miniItems">
+                <a class="hover-text" href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                <a class="hover-text" href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a class="hover-text" href="#">English</a>
+                <a class="hover-text" href="#">My Page</a>
+                <a class="hover-text" href="#">Log out</a>
+            </div>
+        </div>
+        <div class="desktop-menu-container">
+            <img src="icons/interartction-low-resolution-logo-color-on-transparent-background.png" alt="interARTction logo">
+            <ul class="desktop-menu">
+                <li><a class="hover-text" href="home.html" onclick="closeMenu()">Home</a></li>
+                <li><a class="hover-text" href="about.html" onclick="closeMenu()">About</a></li>
+                <li><a class="hover-text" href="#" onclick="closeMenu()">Program</a></li>
+                <li><a class="hover-text" href="#" onclick="closeMenu()">Join</a></li>
+                <li><a class="hover-text" href="#" onclick="closeMenu()">Sponsor</a></li>
+                <li><a class="hover-text" href="#" onclick="closeMenu()">News</a></li>
+                <li><button class="aeCampaign" type="submit">IAEM Campaign</button></li>
+            </ul>
+        </div>
+    </nav>
+    <section class="main-invite-section">
+        <div class="text-container">
+            <div class="intro-text-container">
+                <p class="intro-text1">"Welcome to..."</p>
+                <h1 class="intro-text2">Interdisciplinary Art Exhibition <br>Myanmar <br> 2023</h1>
+            </div>
+            <div class="content-container">
+                <p class="content">
+                    We are delighted to extend our heartfelt invitation to you for an extraordinary event that celebrates the convergence of diverse fields and the evolution of art. <br>
+                    Join us at our Interdisciplinary Exhibition, where the boundaries of creativity are transcended, and new horizons are explored.
+
+                </p>
+            </div>
+            <div class="dtl-container">
+                <p class="date-time">2023.11.1(FRI) ~ 3(SUN)</p>
+                <p class="location">@Peoples' Park, Yangon</p>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ Optional ] I think it would be better if the `hero` section on your `home.html` and `about.html` pages have the `background-image` present on the Behance.net design or a background image of your choice. Currently, it appears that the background is a solid color and not an image which is not consistent with the Behance design. I highly recommend you implement this for your design to mirror the design on Behance.net

| Homepage | Behance design |
|----------------|------------------------|
|![Screenshot from 2023-06-01 02-02-59](https://github.com/Pi1998/first-capstone/assets/16392046/14ea903a-ed23-4d68-bde8-2288066416ba)|![Screenshot from 2023-06-01 02-12-45](https://github.com/Pi1998/first-capstone/assets/16392046/e3675ea3-8038-4892-97a9-670c57b8f33a)|


| About page | Behance design |
|----------------|------------------------|
|![Screenshot from 2023-06-01 02-14-39](https://github.com/Pi1998/first-capstone/assets/16392046/6597b710-1c51-4f78-94cf-2f18531811a9)|![Screenshot from 2023-06-01 02-18-47](https://github.com/Pi1998/first-capstone/assets/16392046/9801fdeb-c9e5-4c60-b138-e12c2e85e9f8)